File: baekjoon/[0] 0-4999/4344_평균은넘겠지.py
N = int(input())

for i in range(N):
    n_list = list(map(int, input().split()))
    del n_list[0]
    sum = 0
    for i in range(len(n_list)):
        sum += n_list[i]
    avg = sum / len(n_list)
    
    student_cnt = [i for i in n_list if i > avg]
    print("{:.3f}%".format(len(student_cnt)/len(n_list)*100))

File: baekjoon/[0] 0-4999/4344_평균은넘겠지.kt
import java.util.*
import java.io.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    var n = readLine().toInt()
    for (i in 1..n){
        var tokenizer = StringTokenizer(readLine())
        var m = tokenizer.nextToken().toInt()
        var intarr = IntArray(m, {0})
        var sum = 0
        var cnt = 0

        for (j in 1..m){
            var a = tokenizer.nextToken().toInt()
            sum += a
            intarr[j-1] = a
        }
        var avg = sum / m.toDouble()
        intarr.forEach{
            if(it.toDouble() > avg) cnt += 1
        }
        println("${String.format("%.3f", (cnt/m.toDouble())*100)}%")
    }
}
